solidity-frontend/src/main/java/org/sonarsource/solidity/frontend/Utils.java

```diff
@@ -97,4 +102,41 @@ public static boolean isCommentSignificant(String token) {
     return matcher.find();
   }
 
+  public static boolean isTernaryExpression(StatementContext ctx) {
+    // TODO
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://sonarcloud.io/organizations/stylianos-agapiou-sonarsource-gi/rules#rule_key=squid%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.sonarsource.solidity%3Asonar-solidity&pullRequest=22&issues=AWQ2SgNkYwZU9vvru9oe&open=AWQ2SgNkYwZU9vvru9oe)

[ISSUE_KEY]: AWQ2SgNkYwZU9vvru9oe